app/views/partials/channels-tab-routes.html

```diff
@@ -1,17 +1,104 @@
+      <label>Primary Route</label><br />
+      <div class="btn-group" style="clear: both;">
+        <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="true">True</label>
+        <label class="btn btn-primary" ng-model="newRoute.primary" btn-radio="false">False</label>
+      </div>
+    </div>
+  </div>
+  <div class="form-group" style="display: inline-block; width: 100%">
+    <div class="form-group" style="float: left;">
+      <label>Route Type</label><br />
+      <div class="btn-group" style="clear: both;">
+        <label class="btn btn-primary" ng-model="newRoute.type" btn-radio="'http'">HTTP</label>
+        <label class="btn btn-primary" ng-model="newRoute.type" btn-radio="'tcp'">TCP</label>
+        <label class="btn btn-primary" ng-model="newRoute.type" btn-radio="'mllp'">MLLP</label>
+      </div>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;">
+      <label>Route Secured</label><br />
+      <div class="btn-group" style="clear: both;">
+        <label class="btn btn-primary" ng-model="newRoute.secured" btn-radio="true">True</label>
+        <label class="btn btn-primary" ng-model="newRoute.secured" btn-radio="false">False</label>
+      </div>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;" ng-if="newRoute.secured==true">
+      <label>Host certificate authority</label><br />
+      <select class="form-control" style="width: auto;" ng-model="newRoute.cert" ng-options="cert._id as cert.commonName for cert in trustedCerts">
+        <option value="">Use default CAs</option>
+      </select>
+    </div>
+  </div>
+  <div class="form-group-2columns">
+    <div class="form-group" style="float: left;" ng-class="{ 'has-error' : ngErrorRoute.host }">
+      <label><span style="color: #a94442;">*</span> Host</label>
+      <input ng-model="newRoute.host" class="form-control" placeholder="Route Host" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</div>
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;" ng-class="{ 'has-error' : ngErrorRoute.port }">
+      <label><span style="color: #a94442;">*</span> Port</label><br />
+      <input ng-model="newRoute.port" class="form-control" placeholder="Route Port" type="text" />
+      <div class="has-error-msg"><i class="glyphicon glyphicon-warning-sign"></i>{{ngErrorRoute.portError}}</div>
+    </div>
+  </div>
+  <div class="form-group-2columns" ng-class="{ 'has-error' : ngErrorRoute.pathTransform }">
+    <div class="form-group" style="float: left;">
+      <label><span style="color: #a94442;">**</span> Route Path</label>
+      <input ng-model="newRoute.path" class="form-control" placeholder="Route Path" type="text" />
+    </div>
+    <div class="form-group" style="float: left; margin-left: 20px;">
+      <label><span style="color: #a94442;">**</span> Route Path Transform</label><br />
```

Comment:
Would it be possible to add a tooltip for the pathTransform? It's a tricky field since you've got to add the s/from/to/g expression so help would definitely be useful